src/main/java/com/envision/Staffing/services/WorkflowService.java

```diff
@@ -85,7 +85,7 @@ private void sendOutput(JobDetails jobDetails, ByteArrayOutputStream outputExcel
 	}
 
 	private void sendOutputToEmail(ByteArrayOutputStream outputExcelData, String email) {
-		emailService.sendMail(email, "WorkflowTest-1", "--Successfull--", outputExcelData);
+		emailService.sendMail(email, "WorkflowTest-1", "--Successfull--<br/>Need to add more details", outputExcelData);
```

Comment:
Change the text to <More details will be added>

Reply:
@naveenkumarkandasamy What should be the text?